vault/router.go

```diff
@@ -522,15 +524,27 @@ func (r *Router) routeCommon(ctx context.Context, req *logical.Request, existenc
 	r.l.RLock()
 	adjustedPath := req.Path
 	mount, raw, ok := r.root.LongestPrefix(ns.Path + adjustedPath)
+	if r.logger.IsTrace() {
```

Comment:
I don't think we should be using the logger `Is*` methods, the only motivation for ever using them is a potential performance micro-optimization.  Also, can we combine these two log statements into one?